screensharing/main.js

```diff
@@ -85,6 +105,62 @@ class ScreenShareMainHook {
             case SCREEN_SHARE_EVENTS.STOP_SCREEN_SHARE:
                 this._jitsiMeetWindow.webContents.send(SCREEN_SHARE_EVENTS_CHANNEL, { data });
                 break;
+            case SCREEN_SHARE_EVENTS.DO_GDM: {
+                const { callback } = this._gdmData;
+
+                this._gdmData = null;
+
+                if (!data.streamId) {
+                    callback(null);
+
+                    break;
+                }
+
+                // The API requires a `DesktopCapturerSource` object. Find the matching one.
+                const constraints = {
```

Comment:
So according to this https://www.electronjs.org/docs/latest/api/session#sessetdisplaymediarequesthandlerhandler-opts we need to provide only id and name. Doesn't this work? 

If it works everything should be way more simpler.

Reply:
It doesn't. It needs the object.

Reply:
Have you tested it? I tested it on MacOS and it works with passing only: 
```
{ video: {id,name} }
```
as described in the docs!

Reply:
I only passed the id... maybe it needs the name too? I'll test!

Reply:
Yep. I was passing the name too and it worked

Reply:
Alright, I had to modify Jitsi Meet to pass around the entire object (why not!) rather than the id alone and it works indeed.